README.md

```diff
@@ -447,6 +448,181 @@ you can create your own Configuration(s) and compose them with Config's ++ opera
 
 Then you can build as usual with CONFIG=MyConfig.
 
+## <a name="debug"></a> Debugging with GDB
+
+### 1) Generating the RBB emulator
+
+The objective of this section is to use GNU debugger to debug programs RISC-V programs running on the emulator in the same fashion as in [Spike](https://github.com/riscv/riscv-isa-sim#debugging-with-gdb).
+
+For that we need to build a Remote Bit Bang enabled emulator, this could be any of the listed configurations in `rocket-chip/src/main/scala/system/Configs.scala`. Basically all we need is to extend our configuration with JtagDTMSystem that will deal with program loading and interface with the debugger. In the following example we added this extention to the DefaultConfig one.
```

Comment:
edited version:

For that we need to add a Remote Bit-Bang client to the emulator. We can do so by extending our `Config` with `JtagDTMSystem`, which will add a `DebugTransportModuleJTAG` to the DUT and connect a `SimJTAG` module in the Test Harness. This will allow OpenOCD to interface with the emulator, and GDB can interface with OpenOCD. In the following example we added this `Config` extension to the `DefaultConfig`:

Reply:
Done